IndyAV/control/indyav_control/CMakeLists.txt

```diff
@@ -68,8 +68,11 @@ add_dependencies(simulated_steering_driver
   std_msgs
   ${catkin_EXPORTED_TARGETS})
 target_link_libraries(simulated_steering_driver
-  ${catkin_LIBRARIES}
-)
+  ${catkin_LIBRARIES})
+add_executable(joy_teleop 
+  src/joy_teleop.cpp)
+target_link_libraries(joy_teleop 
+  ${catkin_LIBRARIES})
```

Comment:
you forgot to add `add_dependencies( joy_teleop ${catkin_EXPORTED_TARGETS})` since your target requires a rosmsg.

Reply:
fixed